pyscriptjs/src/python/pyscript/_plugin.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, name=None):
         self._custom_elements = []
         self.app = None
 
-    def init(self, app):
+    async def init(self, app):
```

Comment:
Is `init` part of the public API for a Python plugin, i.e. are we encouraging users to override/extend it? If so, we should document this change.